pdfly/cli.py

```diff
@@ -240,5 +240,11 @@ def x2pdf(
             writable=True,
         ),
     ],
+    format: str = typer.Option(
+        "A4-portrait",
```

Comment:
The current behaviour should not change: by default not `--format` should be enforced